packages/model/CsvMapping.ts

```diff
@@ -1,6 +1,25 @@
-import { CsvSource } from './CsvSource'
 import { RdfResourceCore } from '@tpluscode/rdfine/RdfResource'
+import { Constructor, property } from '@tpluscode/rdfine'
+import { Collection } from '@rdfine/hydra'
+import { CsvSource } from './CsvSource'
+import { Table } from './Table'
+import { cc } from '@cube-creator/core/namespace'
 
 export interface CsvMapping extends RdfResourceCore {
   sources: CsvSource[]
+  tables: Collection<Table>
```

Comment:
It's a bit weird to have a collection in one case and an array in the other. Is it just because we'll unify the sources in the future?

Reply:
Collection is not a language thing, it's a separate RDF resource.
While the array is simply multiple objects of a property.

We could have tables also directly associated with the mapping resource, not problem there.

Reply:
The only issue I have is naming. It would be less confusing to have either `tables: Table[]` + `tableCollection: Collection<Table>` or change `sources` to also return a collection.

Reply:
To be honest, this should actually be just the identifier because the collection will never be reachable from a CSV mapping's graph. So preferably

```
tableCollection: RdfResource
```

or something like this, which doesn't exist at the moment but add a little strong typing

```
tableCollection: Link<Collection<Table>>
```

Reply:
Ok, I changed exactly like that